<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPrintDialog</class>
 <widget class="QDialog" name="ConfigPrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Configura Listino</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="listinoTableWidget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Colonna</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stretch</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="addListinoButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remListinoButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Configura Ordine</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="ordineTableWidget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Colonna</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stretch</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addOrdineButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remOrdineButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Salva</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remListinoButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>removeColumn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addListinoButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>addColumn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOrdineButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>addColumn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remOrdineButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigPrintDialog</receiver>
   <slot>removeColumn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addColumn()</slot>
  <slot>removeColumn()</slot>
  <slot>save()</slot>
 </slots>
</ui>
